lab_6_pipeline/pipeline.py

```diff
@@ -173,11 +270,26 @@ def __init__(self, corpus_manager: CorpusManager, analyzer: LibraryWrapper) -> N
             corpus_manager (CorpusManager): CorpusManager instance
             analyzer (LibraryWrapper): Analyzer instance
         """
+        self.corpus_manager = corpus_manager
+        self.analyzer = analyzer
 
     def run(self) -> None:
         """
         Visualize the frequencies of each part of speech.
         """
+        articles = self.corpus_manager.get_articles()
+
+        for index, article in articles.items():
+            if (article.get_file_path(kind=ArtifactType.STANZA_CONLLU)
+                    .stat().st_size == 0):
+                raise EmptyFileError
```

Comment:
and you said in docstring that you do not know what this Error is for..

Reply:
now I know)